nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/resources/OSGI-INF/tag-service-properties.xml

```diff
@@ -6,6 +6,13 @@
       Property that enables sanitization on tags.
     </documentation>
     <property name="nuxeo.tag.sanitization.enabled">true</property>
+
+    <documentation>
+      Property that enables tags removal when trashing a document.
+
+      @since 10.10-HF21
```

Comment:
Given that `master` doesn't have this option, shouldn't we mark it deprecated immediately?

Reply:
I agree!